metadata_scripts/GCEMetadataScripts/main.go

```diff
@@ -258,7 +259,13 @@ func getMetadata() (map[string]string, error) {
 }
 
 func getScripts(mdsm map[metadataScriptType]string) ([]metadataScript, error) {
-	md, err := getMetadata()
+	md, err := getMetadata("/instance/attributes")
+	if err != nil {
+		return nil, err
+	}
+	if md == nil {
```

Comment:
This only will work if there are no instance attributes, not just if there are no instance startup scripts, I don't think this is what we want. Looking at the code this may need a decent refactor to handle project and instance scripts, we should probably use the json unmatshaler instead of the iteration and key lookup